scripts/installer_linux/make_deb.sh

```diff
@@ -83,15 +83,21 @@ cp -r "${INDIR}/Surge XT Effects" ${PACKAGE_NAME}/usr/bin/
 # cp -r ../build/surge_products/Surge.lv2 ${PACKAGE_NAME}/usr/lib/lv2/
 
 # set permissions on shared libraries
-find ${PACKAGE_NAME}/usr/lib/vst3/ -type f -iname "*.so" | xargs chmod 0644
```

Comment:
This was barfing on my system due to spaces in the vst3 path, eg "Surge XT". xargs can take the -0 parameter to ignore spaces as delimeters, but I believe using -exec chmod should be more universal.